docs/releases/v2022.1.1.rst

```diff
@@ -0,0 +1,78 @@
+Gluon 2022.1.1
+============
+
+Important notes
+---------------
+
+This release mitigates multiple flaws in the Linux wireless stack causing RCE as well as DoS vulnerabilities.
+
+
+Added hardware support
+----------------------
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+- GL.iNet
+
+  - GL-AP1300
+
+mpc85xx-p1010
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - TL-WDR4900 (v1)
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- ZyXEL
+
+  - NWA50AX
+
+rockchip-armv8
+~~~~~~~~~~~~~~
+
+-  FriendlyElec
+
+   -  NanoPi R2S
+
+Bugfixes
+--------
+
+ * Fixes the update path for GL-AR300M devices.
+ * Multiple mitigations for (`critical vulnerabilities <https://seclists.org/oss-sec/2022/q4/20>`_) in the Linux kernel
```

Comment:
```suggestion
 * Multiple mitigations for (`critical vulnerabilities <https://seclists.org/oss-sec/2022/q4/20>`_) in the Linux kernel WLAN stack. Only affects Gluon v2022.1, older Gluon versions are unaffected.
```

Reply:
This feels like the better place to leave the warning from above. Will fix my previous commit.

Reply:
resolved in 80d219b74d35de93113f1c00d2977529239cdd03